roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -1,8 +1,12 @@
 ---
 # Set to true to allow pre-checks to fail and continue deployment
 ignore_assert_errors: false
+# Set to true to skip 'Stop if even number of etcd hosts' assert check
+# Running an even number of etcd nodes is *not* recommended for production clusters
+skip_etcd_even_nodes_check: false
```

Comment:
Point to consider: since it shouldn't be in inventory, maybe we should put it in vars instead of default so inventory does not override it